packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -201,12 +202,10 @@ export function acceptIncomingCall(incomingCall, spark) {
   return (dispatch) => {
     spark.phone.createLocalMediaStream()
       .then((localMediaStream) => {
+        dispatch(storeCall(incomingCall));
```

Comment:
The reason I didn't store the call here when I initially wrote it was because I wanted to make sure the `.answer` succeeded before storing. We'll need make sure to do more work around handling failing cases of all call statuses.